README.md

```diff
@@ -1 +1,88 @@
-# node-modules-object-lab
\ No newline at end of file
+# Using Object Module
+
+## Objectives
+
+1. Write a simple non-npm module with module.exports = {}
+1. Import your module from another file
+1. Invoke a method from the module
```

Comment:
It seems that students don't actually accomplish this objective. They just `console.log` the second email address.